.github/CONTRIBUTING.md

```diff
@@ -26,7 +26,11 @@ We recommend [using Git](https://docs.github.com/en/get-started/quickstart/set-u
 
 New to contributing? You can practice by contributing to [octocat/Spoon-Knife](https://github.com/octocat/Spoon-Knife) with the help of [this guide](https://docs.github.com/en/get-started/quickstart/contributing-to-projects).
 
-We format our code with Prettier, which makes sure all code follows the same style and makes it look nice and neat. You can install [Prettier for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or enable the "Format code using Prettier" workflow on your fork to have Prettier format your code.
+Looking for a code editor? We recommend [Visual Studio Code](https://code.visualstudio.com/).
+
+We format our code with Prettier, which makes sure all code follows the same style and makes it look nice and neat. You can install [Prettier for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) or enable the "Format code using Prettier" workflow on your fork to have Prettier format your code. If your pull request has formatting issues, don't worry, we can fix it for you.
```

Comment:
we're going to remove the workflow so let's not mention it

Reply:
i mean, you can mention running it manually. we'll probably keep it but without the auto-commit

Reply:
> we're going to remove the workflow so let's not mention it

We are?

Reply:
yeah

Reply:
Like, in the next few months? I kinda like it because formatting issues still pop up in my pushes somewhat frequently (sometimes I use the web UI which doesn't have a format tool, sometimes I just don't think I need to, I haven't been able to get Prettier for VS Code to format CSS, etc.). Having it as an option has been nice. The only equivalent solution I can think of would be through Node.js(?) [like TurboWarp/extensions does](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md#linting-validation-and-formatting).

Reply:
You'll still be able to manually run the auto-format action, if you see that the still automatic "check" action fails

Reply:
Oh! In that case, it _might_ be nice to mention that you can manually dispatch it, then.

Reply:
> we're going to remove the workflow so let's not mention it

Is there a specific reason for this?

I can't say I'd be devastated if it were removed, but it is nice since it corrects random little things that accumulate.

Reply:
If we have a build step (or have at least `package.json`) then we can get away with one script to run it all, otherwise a single workflow on GitHub should be fine.

Reply:
We almost always follow Prettier.